components/automate-ui/src/app/services/sidebar/sidebar.effects.ts

```diff
@@ -94,9 +93,7 @@ export class SidebarEffects {
         eventFeedActions.getInitialFeed(),
         eventFeedActions.getGuitarStrings(),
         eventFeedActions.getTypeCounts(),
-        eventFeedActions.getTaskCounts(),
-        new GetNodes(),
-        new GetNodeCount()
```

Comment:
Stop updates to the chef server and orgs sidebar from updating the nodes and node counts.